extensions/amp-story/1.0/navigation-state.js

```diff
@@ -84,14 +84,17 @@ export class NavigationState {
    * @param {number} pageIndex
    * @param {number} totalPages
    * @param {string} pageId
+   * @param {string} previousPageId
```

Comment:
Nit: `previousPageId` is nullable, for the first navigation (when the story renders)

Reply:
Done

Reply:
Nit: `?string` is the shorthand for `string|null` if you want :))